baseframe/static/css/mui.css

```diff
@@ -3559,7 +3559,6 @@ h4, h5, h6 {
 
 .content-wrapper {
   position: relative;
-  z-index: 1;
   padding-top: 50px;
```

Comment:
Removed z-index of content wrapper div so that the select2 dropdown, which inside the this div can have  an higher z-index than the fixed header